dallinger/command_line/docker.py

```diff
@@ -139,7 +139,8 @@ def push(use_existing: bool, **kwargs) -> str:
 
     config = get_config()
     config.load()
-    _, tmp = setup_experiment(log=log, debug=True, local_checks=False)
+    app_name = kwargs.get("app_name", None)
+    _, tmp = setup_experiment(log=log, debug=True, local_checks=False, app=app_name)
```

Comment:
I might note that `app` and `app_name` is used inconsistently. But I guess this would imply more changes to the code not appropriate for this fix.

Reply:
Yeah it's annoying! At some point it would be good to refactor all that.